src/main/scala/com/redhat/et/silex/feature/extractor/spark/spark.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * This file is part of the "silex" library of helpers for Apache Spark.
+ *
+ * Copyright (c) 2015 Red Hat, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.c
+ */
+
+package com.redhat.et.silex.feature.extractor.spark
+
+import com.redhat.et.silex.feature.extractor.FeatureSeq
+
+import org.apache.spark.mllib.linalg.{
```

Comment:
noted for future reference:  we should really have a convention about what we call Spark/Breeze vectors in Silex